units/user@.service.in

```diff
@@ -12,6 +12,7 @@ Description=User Manager for UID %i
 Documentation=man:user@.service(5)
 After=systemd-user-sessions.service user-runtime-dir@%i.service dbus.service
 Requires=user-runtime-dir@%i.service
+IgnoreOnIsolate=true
```

Comment:
looks ok, but hmm, so far we always wrote "yes" rather than "true" in all the unit files we ship... I figure we should stick to that. Can you change this to "yes" rather than "true"? There's of course no actual change in that, except adherence to a common style, which makes me sleep better at night... ;-)

Reply:
sure :-)